src/lib/Libdb/pgsql/pbs_ds_systemd

```diff
@@ -38,10 +38,10 @@
 
 . ${PBS_CONF_FILE:-/etc/pbs.conf}
 
-is_systemd=0
-systemctl --version >/dev/null 2>&1
-if [ $? -eq 0 ] ; then
-    is_systemd=1
+is_systemd=1
+_status=$(systemctl is-system-running 2>/dev/null)
+if [ "x${_status}" == "xoffline" -o "x${_status}" == "xunknown" ] ; then
```

Comment:
Why are we adding an extra character "x" before the variable string? I see without adding that character also the comparison works fine. Curious to know.

Reply:
Its extra precaution if ${_status} is empty you will compare x == xoffline.

Reply:
Yeah, I thought the same, But with empty string also working fine. 
#!/bin/bash

var1="same"
var2=
if [ "$var1" == "$var2" ]
then
	echo "Equal"
else
	echo "Not Equal"
fi


Did you see any error with this ?

Reply:
I don't see any harm in adding that check. If you see it's having any specific reason, we will add that character.